doc/policies/admin.rst

```diff
@@ -193,21 +197,31 @@ which have a minimum length of 8.
 The logic of the ``otp_pin_contents`` can be enhanced and reversed using the
 characters ``+`` and ``-``.
 
-``-cn`` would still mean, that the OTP PIN needs to contain letters and digits
-and it must not contain any other characters.
+``-cn`` (denial)
+
+   The PIN must not contain a character and must not contain a number.
+   *test1234* would not be a valid PIN, since it does contains numbers and characters.
+   *test///* would not be a valid PIN, since it contains numbers.
 
-``-cn`` (substraction)
+``-s`` (denial)
 
-   *test1234* would be a valid OTP PIN, but *test12$$* and *testABCS* would
-   not be valid OTP PINs. The later since it does not contain digits, the first
-   (*test12$$*) since it does contain a special character ($), which it should not.
+   The PIN must not contain a special character.
+   **test1234* would be a valid PIN. *test12$$* would not.
 
 ``+cn`` (grouping)
 
    combines the two required groups. I.e. the OTP PIN should contain
    characters from the sum of the two groups.
    *test1234*, *test12$$*, *test*
    and *1234* would all be valid OTP PINs.
+   Note, how this is different to ``-s``.
```

Comment:
Yes. And it allows special characters to be included. I added:

~~~~
Note, how this is different to ``-s``, since it allows special characters to be
   included.
~~~~

Reply:
Added some more text in f79cae4b